components/formats-gpl/matlab/bfCheckJavaPath.m

```diff
@@ -77,9 +77,9 @@
 if status
     % Read Bio-Formats version
     if is_octave()
-        version = char(java_get('loci.formats.FormatTools', 'VERSION'));
+        version = java_get('loci.formats.FormatTools', 'VERSION');
     else
-        version = char(loci.formats.FormatTools.VERSION);
+        version = loci.formats.FormatTools.VERSION;
```

Comment:
hmmm... that's interesting. I can't test this on Matlab myself but seems to go against their documentation. Maybe what they meant to say is that Matlab does an implicit conversion to char when required and not as soon as the java method returns? (their documentation does state "When a method call returns data of type java.lang.Object, MATLAB converts its value").

Anyway, this commit is not really important, I just thought of removing unnecessary function calls. The other does cause compatibility issues for Octave though.